draft-ietf-masque-h3-datagram.md

```diff
@@ -211,14 +211,14 @@ avoid "sticking out"; see {{security}}.
 \[\[RFC editor: please remove this section before publication.]]
 
 Some revisions of this draft specification use a different value (the Identifier
-field of a Setting in the HTTP/3 SETTINGS frame) for the H3_DATAGRAM Settings
-Parameter. This allows new draft revisions to make incompatible changes.
-Multiple draft versions MAY be supported by sending multiple values for
-H3_DATAGRAM. Once SETTINGS have been sent and received, an implementation that
-supports multiple drafts MUST compute the intersection of the values it has sent
-and received, and then it MUST select and use the most recent draft version from
-the intersection set. This ensures that both peers negotiate the same draft
-version.
+field of a parameter in the HTTP/3 SETTINGS frame) for the SETTINGS_H3_DATAGRAM
```

Comment:
This was intentional to match the wire format diagrams from section 7.2.4 or the H3 draft
```suggestion
field of a Setting in the HTTP/3 SETTINGS frame) for the SETTINGS_H3_DATAGRAM
```

But I don't care strongly, this'll get removed before publication

Reply:
happy to stick with old text, I hope the text is moot now